### ISSUE TITLE
Passing document to .use

### ISSUE THREAD

**Issue by nwhite89** [webpack-contrib/style-loader#44]
I need to be able to pass the document to the `.use` method in which it should attach the document, due to iFrame buster business. Any way of doing this currently or thought on perhaps adding as an added feature?

Thanks.

**Comment by solsson** [comment]
+1
```javascript
var style = require("style/useable!css!./embed.css");
$('#myIframe')[0].onload = function() {
  // TODO add style to this frame, which is same origin
};
```

**Comment by michael-ciniawsky** [comment]
Related to #135

**Comment by michael-ciniawsky** [comment]
Closed by #135
